problems/make_it_molecular/verify.js

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env node
+
+var exec = require('child_process').exec;
+var fs = require('fs');
+var path = require('path');
+var filename = "01-item.mustache";
+
+//check the directory exists
+//check the file exists
+//verify the contents of the file
+
+findFile();
+
+function findFile() {
+  if (process.cwd().match("molecules")) {
+    check(process.cwd())
+  } else {
+    check(path.join(process.cwd(), "/molecules/"))
+  }
+
+  function check(userspath) {
+    fs.readdir(userspath, function(err, files) {
+      if (err) return console.log(err);
```

Comment:
I'm not quite sure how to best update these to be more verbose...

A lot of them exist/are structured that way so the workshopper can check things incrementally/match the console logs, and the node.js docs recommend including "if (err) return console.log(err);" when working with fs and files or data.